<launch>
  
  <!-- Start the Loosely Coupled EKF node -->
    <node pkg="loosely_coupled_ekf" type="loosely_coupled_ekf_node" name="loosely_coupled_ekf" output="screen" required="true">
      <param type="double" name="std_deviation_pos" value="0.003" />
      <param type="double" name="std_deviation_vel" value="0.002" />
      <param type="double" name="std_deviation_acc" value="2.000163151535074" />
      <param type="double" name="std_deviation_acc_bias" value="0.00002000163151535074" />
      <param type="double" name="std_deviation_gyr" value="0.040003263030701" />
      <param type="double" name="std_deviation_gyr_bias" value="0.0000004000326303070147" />
      <param type="double" name="std_deviation_pos_i" value="0.003" />
      <param type="double" name="std_deviation_vel_i" value="0.002" />
      <param type="double" name="std_deviation_att_i" value="0.01" />
      <param type="double" name="std_deviation_acc_bias_i" value="2.000163151535074" />
      <param type="double" name="std_deviation_gyr_bias_i" value="0.040003263030701" />
      <rosparam param="initial_pos"> [0.0,0.0,0.0] </rosparam>
      <rosparam param="initial_vel"> [0.0,0.0,0.0] </rosparam>
      <rosparam param="initial_att"> [0.0,0.0,2.356] </rosparam>
      <rosparam param="initial_bias"> [0.,0.,0.,0.,0.,0.] </rosparam>
      <param name="dt_integration" value="0.01" />

      <!-- IF GPS_SENSOR = 0, THEN NOVATEL -->
      <!-- 				       = 1, THEN UBLOX -->
      <param name="GPS_SENSOR" value="1" />
      <!-- IF IMU_SENSOR = 0, THEN XBOW440 -->
      <param name="IMU_SENSOR" value="0" />

      <!-- IF NAV_FRAME = 0, THEN ENU -->
      <!--              = 1, THEN NED -->
      <param name="NAV_FRAME" value="0" />

      <!-- IF POS_MEASURE = true, THEN receiving position measurements -->
      <param name="POS_MEASURE" value="true" />

      <!-- IF VEL_MEASURE = true, THEN receiving velocity measurements -->
      <param name="VEL_MEASURE" value="true" />
    </node>

</launch> 